src/components/utils/redux.ts

```diff
@@ -9,7 +9,9 @@ export const buildCaptureStateKey = (payload: {
 }): Exclude<keyof CaptureState, 'takesHistory'> => {
   const { method, side, variant } = payload
 
+  if (method === 'poa') return 'poa'
```

Comment:
let me suggest a map here

Reply:
I'm not sure I understand how to implement the suggestion of a map here. Currently, only the values of `poa` and `face` directory correspondent to a `method`. But, other methods depend on the values of `side` and `variant`.